src/main/java/com/capitalone/dashboard/controller/DefaultWhiteSourceController.java

```diff
@@ -66,11 +66,15 @@ public ResponseEntity<String> refresh(@Valid @RequestBody WhiteSourceRefreshRequ
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Must provide projectToken or altIdentifier");
         }
 
-        defaultWhiteSourceClient.refresh(request.getProjectToken(), request.getAltIdentifier());
-        String res = StringUtils.isNotEmpty(request.getProjectToken()) ? "projectToken="+request.getProjectToken() : "altIdentifier="+request.getAltIdentifier();
+        String res = "Updated Whitesource component:: ";
+        int refreshCount = defaultWhiteSourceClient.refresh(request.getProjectToken(), request.getAltIdentifier());
+        if (refreshCount > 0) {
+            res += StringUtils.isNotEmpty(request.getProjectToken()) ? "projectToken=" + request.getProjectToken() : "altIdentifier=" + request.getAltIdentifier();
+        }
+        else {res = "No Refresh :: Whitsource component not found";}
```

Comment:
This is nitpicky, but can you change wording to  "Could not refresh: Whitesource component not found" ?